public_html/wp-content/plugins/wordcamp-organizer-survey/wordcamp-organizer-survey.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Plugin Name:     WordCamp Organizer Survey
+ * Plugin URI:      https://wordcamp.org
+ * Description:     Send survey to WordCamp organizers.
+ * Author:          WordCamp.org
+ * Author URI:      https://wordcamp.org
+ * Version:         1
+ *
+ * @package         WordCamp\OrganizerSurvey
+ */
+
+namespace WordCamp\OrganizerSurvey;
+
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\add_page as add_debrief_survey_page;
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\delete_page as delete_debrief_survey_page;
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\add_email as add_debrief_survey_email;
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\delete_email as delete_debrief_survey_email;
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Cron\delete_temp_attendee as delete_debrief_survey_temp_attendee;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Local dependencies.
+ */
+require_once get_includes_path() . 'debrief-survey/email.php';
+require_once get_includes_path() . 'debrief-survey/page.php';
+
+/**
+ * Plugin deactivation hook.
+ */
+register_deactivation_hook( __FILE__, __NAMESPACE__ . '\deactivate' );
+
+/**
+ * Actions & hooks
+ */
+add_action( 'plugins_loaded', __NAMESPACE__ . '\load' );
+
+/**
+ * Get the ID of the survey feature.
+ */
+function get_feature_id() {
+	return 'organizer_survey';
+}
+
+/**
+ * Include the rest of the plugin.
+ *
+ * @return void
+ */
+function load() {
+	// We only want admin panel on central, nothing else.
+	if ( WORDCAMP_ROOT_BLOG_ID === get_current_blog_id() ) {
+		require_once get_includes_path() . 'admin-page.php';
+	}
+
+	if ( is_wordcamp_type( 'next-gen' ) ) {
+		// Debrief Survey.
+		require_once get_includes_path() . 'debrief-survey/cron.php';
+
+		add_action( 'init', __NAMESPACE__ . '\activate_on_current_site' );
+	}
+}
+
+/**
+ * The activation routine for a single site.
+ *
+ * @return void
+ */
+function activate_on_current_site() {
+	// Debrief Survey.
+	add_debrief_survey_page();
+	add_debrief_survey_email();
+
+	// Flushing the rewrite rules is buggy in the context of `switch_to_blog`.
+	// The rules will automatically get recreated on the next request to the site.
+	delete_option( 'rewrite_rules' );
```

Comment:
Ah, yeah. You're right. This should be removed. Previously, when I tried to add the Organizer Reminder, I used `switch_to_blog`, so it's been there ever since.